dashboard/app/controllers/sms_controller.rb

```diff
@@ -32,8 +32,12 @@ def send_sms_link(link, phone)
   end
 
   def send_sms(body, phone)
+    # If the Twilio WebService is unavailable or experiencing latency issues we can no-op this method to avoid
+    # tie-ing up all puma worker threads waiting for the Twilio API to respond by switching a Gatekeeper flag.
+    head :ok unless Gatekeeper.allows('twilio', default: true)
```

Comment:
should this be `return head` to prevent the code following it from being executed? That seems to be what the [documentation example](https://api.rubyonrails.org/classes/ActionController/Head.html#method-i-head) suggests.

Reply:
Yes. Thanks for catching that. Fixed.